miio/integrations/zhimi/humidifier/airhumidifier_miot.py

```diff
@@ -6,14 +6,16 @@
 
 from miio import DeviceStatus, MiotDevice
 from miio.click_common import EnumType, command, format_output
+from miio.devicestatus import sensor, setting
 
 _LOGGER = logging.getLogger(__name__)
 
 
 SMARTMI_EVAPORATIVE_HUMIDIFIER_2 = "zhimi.humidifier.ca4"
+SMARTMI_EVAPORATIVE_HUMIDIFIER_3 = "zhimi.humidifier.ca6"
 
 
-_MAPPINGS = {
+_MAPPINGS_CA4 = {
```

Comment:
It would be better to have the new mappings contained in the very same `_MAPPINGS` as then it will be automatically used based on the model information. Could you change that?

Reply:
I tried to have all mapping in one `_MAPPING`, but in this case I didn't manage to make the whole thing work. I still don't understand why. I'm not a professional software developer anymore ;)

Reply:
Okay, great, let's get this merged then! :-)